RecoTracker/TkHitPairs/interface/LayerHitMapCache.h

```diff
@@ -37,8 +37,13 @@ class LayerHitMapCache {
         resize(other.theContainer.size());
 
       for (size_t i = 0, size = other.theContainer.size(); i != size; ++i) {
-        assert(get(i) == nullptr);               // We don't want to override any existing value
-        theContainer[i].reset(*(other.get(i)));  // pass by reference to denote that we don't own it
+        assert(get(i) == nullptr);  // We don't want to override any existing value
+        auto v = other.get(i);      // pass by reference to denote that we don't own it
```

Comment:
Should this comment be moved to L43 now?

Reply:
done